<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="imageIcon">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFolder">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>170</y>
     <width>142</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择图像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>290</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>··拖拽图像··</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOutput">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>345</y>
     <width>142</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存至输出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editOutput">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>420</y>
     <width>551</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>385</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>输出目录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChoose">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>420</y>
     <width>73</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>folder.ico</normaloff>folder.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_android">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>345</y>
     <width>142</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存至Android工程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_ios">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>345</y>
     <width>142</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存至iOS工程</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_ios">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>5</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>iOS图标</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="imageIcon_ios">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>28</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_ios_2">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>5</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Android图标</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="imageIcon_android">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>28</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>478</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(70, 148, 194);</string>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>233</y>
     <width>142</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清除日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>50</y>
     <width>142</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>刷新图标</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
